twisted/python/_release.py

```diff
@@ -58,8 +58,8 @@ def runCommand(args, cwd=None):
     @raise CommandFailed: when the program exited with a non-0 exit code.
     """
     process = Popen(args, stdout=PIPE, stderr=STDOUT, cwd=cwd)
-    stdout = process.stdout.read()
-    exitCode = process.wait()
+    stdout = process.communicate()[0]
+    exitCode = process.poll()
```

Comment:
I think this entire function should be removed and replaced with [`subprocess.check_output`](https://docs.python.org/2/library/subprocess.html#subprocess.check_output).  It does the same thing, but is only available in Pythons after 2.7.  Fortunately that's all Twisted supports now.